README.md

```diff
@@ -91,7 +91,10 @@ output {
    }            
 }               
 ```
+### ecs_compatibility
+ECS compatibility for V8 was added in 1.3.0. For more details on ECS support refer to this [documentation](docs/ecs_compatibility.md).
```

Comment:
Nit: Reword it Version `1.3.0`. 
Also, I noticed in the [release notes](https://github.com/opensearch-project/logstash-output-opensearch/releases) for version `1.3.0` there is no mention of ECS compatibility but we're mentioning here it was added in that version. We should update the release notes for consistency.